torch/nn/init.py

```diff
@@ -466,6 +466,9 @@ def orthogonal_(tensor, gain=1):
         raise ValueError("Only tensors with 2 or more dimensions are supported")
 
     rows = tensor.size(0)
+    if rows == 0:
```

Comment:
nit: could we use `tensor.numel()` here to also catch the cases where other dimensions are of size 0

Reply:
Hey @albanD, thanks for the review. I've updated the code; is this in line with what you had in mind?

Reply:
Yes!